src/system/device.js

```diff
@@ -225,6 +234,15 @@
                                  me.device.wp ||
                                  me.device.BlackBerry ||
                                  me.device.Kindle || false;
+
+            // Wechat small game platform
+            me.device.isWxSmallGame = (typeof wx !== "undefined");
+
+            if (me.device.isWxSmallGame)
+            {
+                me.device.WxSmallGame = wx;
+            }
+
```

Comment:
we can just check `me.device.isWxSmallGame` around to see if we are running on the weixin platform, no need to create another `me.device.WxSmallGam` alias to `wx` ?